zanata-server/Dockerfile

```diff
@@ -1,23 +1,29 @@
 FROM jboss/wildfly:10.0.0.Final
+MAINTAINER "Ding-Yi Chen" <dchen@redhat.com>
+
+EXPOSE 8080
+# Version should be assigned in build time
+ARG ZANATA_VERSION=3.9.1
```

Comment:
You should add a comment here, since `zanata-server/runapp.sh` now depends on being able to extract this arg value. For instance, if someone added extra whitespace, the regex might break.